### PR TITLE
the assigning items module has been added

### PR DESCRIPTION
**Pull request by Sharukap**

### CODE REVIEW

File: modules/General/ApprovalItem/Http/Controllers/ApprovalItemController.php [hunk 1]
```diff
@@ -14,11 +19,101 @@ public function home()
...
     {
         $items = Process_Item::where('created_by_user_id', '=', Auth::user()->id)->get();
         return view('approvalItem::requests', [
             'items' => $items,
         ]);
     }
+
+    public function choose_assign_staff($id)
+    {
+        //dd($id);
```
Comment (yashodgayashan) [inline modules/General/ApprovalItem/Http/Controllers/ApprovalItemController.php]:
```suggestion
```

File: modules/General/ApprovalItem/Http/Controllers/ApprovalItemController.php [hunk 2]
```diff
@@ -14,11 +19,101 @@ public function home()
         return view('approvalItem::home', compact('name'));
     }
 
+    
+    public function confirm_assign_staff($id,$pid)
+    {
+
```
Comment (yashodgayashan) [inline modules/General/ApprovalItem/Http/Controllers/ApprovalItemController.php]: Please remove unnecessary spacings

File: modules/General/ApprovalItem/views/StaffItemsView.blade.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                <td>{{$row['remark']}}</td>
+                <td>Crime report</td>
+                <td>{{$row['created_at']}}</td>
+                <td><a href='/check/{{ $row['id'] }}' class="btn btn-outline-warning" role="button" >check</a></td>
+                </tr>
+                @endforeach
+        </tbody>
+    </table>
+</div>
+@endsection
```
Comment (yashodgayashan) [inline modules/General/ApprovalItem/views/StaffItemsView.blade.php]:
```suggestion
@endsection

```

File: modules/General/ApprovalItem/views/crimeAssign.blade.php [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+            <tbody>
+                <tr>
+                    <td>{{$crime->id}}</td>
+                    <td>{{$crime->crime_type}}</td>
+                    <td>{{$crime->description}}</td>
+                    <td>{{$crime->polygon}}</td>
+                    <td>{{$crime->created_at}}</td>
+                    <td>{{$crime->action_taken}}</td>
+                    <td>{{$crime->status}}</td>
+                    <!-- <td><a href='/edit/{{ $crime->id }}' class="btn btn-outline-warning" role="button" >Edit</a></td>
```
Comment (yashodgayashan) [inline modules/General/ApprovalItem/views/crimeAssign.blade.php]: Could you please remove those

File: modules/General/ApprovalItem/views/developAssign.blade.php [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+                    <td>{{$user->name}}</td>
+                    <td>{{$user->email}}</td>
+                    <td><a href="/approval-item/confirmassign/{{$user->id}}/{{$Process_item->id}}" class="text-muted">assign</a></td>
+                </tr>
+                @endforeach
+            </tbody>
+        </table>
+    </div>
+</div>
+@endsection
```
Comment (yashodgayashan) [inline modules/General/ApprovalItem/views/developAssign.blade.php]:
```suggestion
@endsection

```

**Review by yashodgayashan — COMMENTED** [review]
remove unnecessary spacings and debug codes.

**Comment by Sharukap** [comment]
@yashodgayashan since we have to show a demo tomorrow we kind of merged these changes manually to boji's repo and adjusted the conflicts so for now keep this on hold and not merge since the changes will be there from Boji's pull request

**Review by yashodgayashan — APPROVED** [review]

**Pull request merged**